packages/ckb/src/utils/cell-dep.ts

```diff
@@ -210,3 +202,58 @@ export const fetchTypeIdCellDeps = async (
 
   return cellDeps;
 };
+
+const VERCEL_STATIC_COMPATIBLE_XUDT_URL = 'https://typeid-contract-cell-deps.vercel.app/compatible-udt.json';
+const GITHUB_STATIC_COMPATIBLE_XUDT_URL =
+  'https://raw.githubusercontent.com/utxostack/typeid-contract-cell-deps/main/compatible-udt.json';
+
+/**
+ * The `CompatibleXUDTRegistry` class is responsible for managing a cache of compatible XUDT (eXtensible User-Defined Token) scripts.
+ * It fetches and caches the compatible tokens from specified URLs and refreshes the cache periodically.
+ */
+export class CompatibleXUDTRegistry {
+  private static cache: CKBComponents.Script[] = [];
+  private static lastFetchTime: number = 0;
+  private static CACHE_DURATION = 3 * 60 * 1000; // 3 minutes ([about 24 CKB blocks](https://docs-old.nervos.org/docs/essays/tx-confirmation))
+  private static xudtUrl = VERCEL_STATIC_COMPATIBLE_XUDT_URL;
+
+  // If you want to get the latest compatible xUDT list, CompatibleXUDTRegistry.refreshCache should be called first
+  static getCompatibleTokens(): CKBComponents.Script[] {
+    const now = Date.now();
+    if (this.cache.length > 0 || now - this.lastFetchTime > this.CACHE_DURATION) {
```

Comment:
The condition `this.cache.length > 0` will refresh the cache every time.


```suggestion
    if (this.cache.length === 0 || now - this.lastFetchTime > this.CACHE_DURATION) {
```

Reply:
Fixed in https://github.com/utxostack/rgbpp-sdk/pull/297/commits/47c7e8d642892c267c15f8ff122e5e0e010c5e30